utils/kata-manager.sh

```diff
@@ -720,26 +721,74 @@ handle_containerd()
 	containerd --version
 }
 
+handle_docker()
+{
+	{ containerd_installed; ret=$?; } || true
+	if [ "$ret" -eq 0 ]
+	then
+		info "Backing up previous containerd configuratinn"
+		local cfg="/etc/containerd/config.toml"
+
+		sudo mv $cfg $cfg.system-$(date -Iseconds)
```

Comment:
This fails on my test box as the file doesn't exist, so please can you add a check:

```suggestion
		[ -e "$cfg" ] && sudo mv $cfg $cfg.system-$(date -Iseconds)
```